src/app/drive/views/RequestAccess/RequestAccess.tsx

```diff
@@ -7,9 +7,14 @@ import { useTranslationContext } from 'app/i18n/provider/TranslationProvider';
 import Button from 'app/shared/components/Button/Button';
 import bigLogo from 'assets/icons/big-logo.svg';
 import './RequestAccess.scss';
+import { logOut } from '../../../auth/services/auth.service';
 
```

Comment:
Maybe we can extract this: `const alertUsersTextareaMaxLenght = 950;` as `const TEXTAREA_MAX_LENGTH = 950;` here.